cleanlab/multilabel_classification/__init__.py

```diff
@@ -0,0 +1,4 @@
+from .rank import get_label_quality_scores
```

Comment:
Did we decide this ok or not? @elisno

Reply:
This behavior was introduced in #542. I believe it was intended then.

Currently (on master), `get_label_quality_scores` is automatically imported with `cleanlab`:

```python
>>> import cleanlab
>>> cleanlab.multilabel_classification.get_label_quality_scores
<function get_label_quality_scores at 0x7f0767244c10>
```

---

I think backward compatibility is maintained by doing this.

Reply:
Note that I added a few extra helper functions to decrease code duplication.

Unless we move those elsewhere (thoughts @jwmueller?), we may need to state explicitly what is accessible from  `cleanlab explicitly.multilabel_classification`.

Reply:
Fine to keep helpers here as long as they're internal methods!